docs/compression_algorithms/Binarization.md

```diff
@@ -36,12 +36,12 @@ Training binarized networks requires special scheduling of the training process.
             "lr_poly_drop_duration_epochs": 30,  // Duration, in epochs, of the learning rate dropping process.
             "disable_wd_start_epoch": 60  // Epoch to disable weight decay in the optimizer
         },
-    
-        "ignored_scopes": ["ResNet/Linear[fc]",
-                           "ResNet/Conv2d[conv1]",
-                           "ResNet/Sequential[layer2]/BasicBlock[0]/Sequential[downsample]",
-                           "ResNet/Sequential[layer3]/BasicBlock[0]/Sequential[downsample]",
-                           "ResNet/Sequential[layer4]/BasicBlock[0]/Sequential[downsample]"]
+
+        "ignored_scopes": ["ResNet/NNCFLinear[fc]/linear_0",
```

Comment:
@vshampor Just to clarify. Does the original graph contain NNCFModules, If the user wrap the model by NNCFNetwork?